services/pom.xml

```diff
@@ -50,6 +50,11 @@
             <artifactId>${keycloak.crypto.artifactId}</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.ibm.icu</groupId>
+            <artifactId>icu4j</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Can you please elaborate why you choose provided here?  With this setting, the JAR would not be packaged.